modules/upgrade/pages/client-migration.adoc

```diff
@@ -1,184 +1,153 @@
 
-image::sles_old_details_page.png[scaledwidth=80%]
 
-+
-If there are menu:Software Updates Available[] in the menu:System Status[] notification install these updates first to avoid trouble during the migration process.
+Traditional clients on earlier {slea}{nbsp}12 Service Packs can be upgraded to later {slea}{nbsp}12 Service Packs in the {productname} {webui}.
 
-. On the system's detail page select the menu:Software[] tab, then the menu:SP Migration[] tab.
-+
+For example, upgrading from {slea}{nbsp}12{nbsp}SP1 to {slea}{nbsp}12{nbsp}SP4.
 
-image::sles_old_details_spmigration.png[scaledwidth=80%]
+Before you begin the migration, make sure that the migration target channels are completely mirrored.
+For the upgrade to {sle}{nbsp}12 SP4, at least the [systemitem]``SLES12-SP4-Pool`` base channel with the [systemitem]``SLE-Manager-Tools12-Pool`` child channel for your architecture is required.
+The matching update channels such as [systemitem]``SLE-Manager-Tools12-Updates`` and [systemitem]``SLES12-SP4-Updates`` are recommended.
 
-. From this tab you will see the installed products listed on your client. Select the wanted menu:Target Products[] (if there is more than one), which is [systemitem]``SUSE Linux Enterprise Server 12{nbsp}SP4``.
-+
+// TODO: Check this:
+You can monitor the mirroring progress in [path]``/var/log/rhn/reposync/sles15-pool-x86_64.log``.
 
-image::sles_migration_target.png[scaledwidth=80%]
 
-+
-Then confirm with menu:Select Channels[].
-+
-
-image::sles_migration_channels.png[scaledwidth=80%]
-. Select menu:Schedule Migration[], and then menu:Confirm[].
-+
+.Procedure: Migration
+. In the {productname} Server {webui}, navigate to menu:Main Menu[Systems] and select the client to be upgraded.
+. If the [guimenu]``System Details`` page has outstanding software updates, install the updates.
+. Navigate to the menu:Software[SP Migration] tab, select [systemitem]``SUSE Linux Enterprise Server 12{nbsp}SP4``, and click btn:[Select Channels].
+. Select the mandatory child channels, and any optional child channels.
+Click btn:[Schedule Migration].
+. Choose a time for the migration, and click btn:[Confirm].
+You can also perform a dry run of the migration to check that it will be successful.
 
 image::sles_migration_schedule.png[scaledwidth=80%]
 
-Check the menu:System Status[] on the system's details when the migration is done.
-
-image::sles_migrated.png[scaledwidth=80%]
+You can check the progress of the migration by navigating to menu:System Details[System Status].
 
-If the menu:System Status[] notification does not report a successful migration but lists menu:Software Updates Available[], install the update now and then check again.
+If the system status reports that the migration has failed, but also shows that software updates are available, install the update and check the status again.
 
-Finally, consider to schedule a reboot.
+After the migration is complete, reboot the client.
```

Comment:
We must make sure that users will get this right.  The user must not have the impression that he has to do this client-side. He should schedule a reboot in SUMA.